README.md

```diff
@@ -1,2 +1,32 @@
-# sdk
-Instructions and supporting files for SDK generation
+# PDC SDK Generators
+
+This repository houses the infrastructure for generating PDC SDKs.
+
+We're using a project called [swagger-codegen](https://github.com/swagger-api/swagger-codegen) to power our SDK, and this repository provides tooling and templates
+to make sdk generation easier in particular target languages.  The codegen project supports a [many languages](https://github.com/swagger-api/swagger-codegen/tree/master/modules/swagger-codegen/src/main/resources)
```

Comment:
```suggestion
to make sdk generation easier in particular target languages. The codegen project supports a [many languages](https://github.com/swagger-api/swagger-codegen/tree/master/modules/swagger-codegen/src/main/resources)
```

Reply:
(This is not a real suggestion, so much as a poetic lament.)

Reply:
I do what prettier tells me to do justin, your wish is granted.

Reply:
😮 Prettier removed the double-spaces?? I LOVE PRETTIER

Reply:
It did in the other repo -- https://github.com/PhilanthropyDataCommons/service/pull/695#discussion_r1443550634

(we aren't running it here, but I try to abide by the demand of automated tools in all aspects of life)

Reply:
I love that Prettier has more credibility than _me_.